src/xss-common-includes.h

```diff
@@ -47,14 +48,17 @@
  * Force inline in cygwin to work around a compiler bug. See
  * https://github.com/numpy/numpy/pull/22315#issuecomment-1267757584
  */
+#define X86_SIMD_SORT_INLINE_ONLY inline
```

Comment:
This makes the comment above misplaced as it applies to the define below

Reply:
Ah, good point. Will fix the order :)